device_authorize_request_handler.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package fosite
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/ory/fosite/i18n"
+	"github.com/ory/x/errorsx"
+)
+
+func (f *Fosite) NewDeviceAuthorizeRequest(ctx context.Context, r *http.Request) (DeviceAuthorizeRequester, error) {
+	request := NewDeviceAuthorizeRequest()
+	request.Lang = i18n.GetLangFromRequest(f.Config.GetMessageCatalog(ctx), r)
+
+	if err := r.ParseForm(); err != nil {
+		return nil, errorsx.WithStack(ErrInvalidRequest.WithHint("Unable to parse HTTP body, make sure to send a properly formatted form request body.").WithWrap(err).WithDebug(err.Error()))
+	}
+	request.Form = r.Form
+
+	verifier := request.GetRequestForm().Get("device_verifier")
```

Comment:
Hi, may I know what `device_verifier` is? I cannot find it in https://www.rfc-editor.org/rfc/rfc8628

Reply:
I don't remember what this code is for ? Maybe @BuzzBumbleBee knows ?

BTW, I've wrote some sample clients to test it here:
https://github.com/supercairos/oauth-device-flow-client-sample